python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -169,6 +169,14 @@ def runApPipeGen3(workspace, parsedCmdLine, processes=1):
     pipelineArgs.extend(["--processes", str(processes)])
     pipelineArgs.extend(["--register-dataset-types"])
 
+    # TODO: this config option resets the skymap dataset type to the templated
+    # name used by default prior to DM-27147.  After RFC-471 is adopted, the
+    # ap_verify test data should be migrated to use the new dataset type name
+    # (and collection naming convention) and this config override can be
+    # removed (other changes will probably be necessary as well, to reflect
+    # changes to the collection naming conventions).
```

Comment:
I suggest simplifying this to "TODO: remove after ap_verify datasets follow RFC-471 conventions".